README.md

```diff
@@ -56,8 +56,90 @@ Clean up the Kubernetes/OpenShift cluster:
 ```shell script
 $ make cluster/clean
 ```
+### Deploy the Operator in cluster
 
-## VPC Peering 
+Build and push the image
+```shell script
+$ make image/push IMAGE_ORG=<your quay org>
+```
+
+Update the `deploy/operator.yaml` to point at the newly built image
+```
+      ...
+        - name: cloud-resource-operator
+          image: quay.io/<your quay org>/cloud-resource-operator:v0.7.1
+          command:
+          - cloud-resource-operator
+          imagePullPolicy: Always
+      ...
+```
+
+Seed the Kubernetes/OpenShift cluster with required resources:
+```shell script
+$ make cluster/prepare
+```
+
+Create the service account
+```shell script
+$ make setup/service_account
+```
+
+Deploy the operator
+```shell script
+$ oc apply -f deploy/operator.yaml
+```
+
+### Via the Operator Catalog
+
+*Note* the following steps to installing CRO via the Operator Catalog are intended for Integreatly team members and testing changes in development
+
+Build and push the image to your Quay Org
+```shell script
+$ make image/push IMAGE_ORG=<your quay org>
+```
+
+Push the manifests to your Quay Org
+```shell script
+$ make manifest/push IMAGE_ORG=<your quay org>
+```
+
+Create a new operator source in your Openshift Cluster, updating the `registryNamespace` to your own org name.
+```
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: integreatly-operators
+  namespace: openshift-marketplace
+spec:
+  authorizationToken: {}
+  displayName: Integreatly Operators
+  endpoint: 'https://quay.io/cnr'
+  publisher: Integreatly Publisher
+  registryNamespace: <<your quay org name>>
+  type: appregistry
+```
+Once the OLM has reconciled, the Cloud Resource Operator should be available in the Catalog.
```

Comment:
You mentioned `CRO` earlier, maybe we can refer to it once as `Cloud resource operator (CRO)` and then as `CRO` afterwards